using_images/db_images/mysql.adoc

```diff
@@ -208,25 +231,30 @@ $ oc create -f \
 ----
 ====
 
+The following sections detail the objects defined in the example template and
+describe how they work together to start a cluster of MySQL servers implementing
+master-slave replication. This is the recommended replication strategy for
+MySQL.
```

Comment:
@mfojtik Was there a reason to leave this paragraph out in your initial PR? I added it in here (from the Mongo topic), and dropped the "automated failover" bit, as it seemed like that wasn't applicable here. Lmk either way.